pyInterface/module/_integrals.py

```diff
@@ -30,7 +30,14 @@ def calcIntegrals(integralFileName, eventAndAmpFileDict, multiBin, weightFileNam
 				pyRootPwa.utils.printErr("could not read metadata from amplitude file '" + ampFileName + "'. Aborting...")
 				return False
 			ampMetas.append(ampMeta)
-			integralMetaData.addKeyFileContent(ampMeta.keyfileContent())
+
+			if len(integrals) == 1: # first eventFieldID of the current bin
+				if not integralMetaData.addKeyFileContent(ampMeta.keyfileContent()):
+					pyRootPwa.utils.printWarn("could not add the keyfile content.")
```

Comment:
`addKeyFileContent` already prints a similar error message, so I think this could be skipped here.
Instead a `return false` should be added to signal this error.